ledfx/tools/ts_generator.py

```
@@ -0,0 +1,858 @@
+    # Custom Validators first
+    if hasattr(validator, "__name__"):
+        validator_name = validator.__name__
+        if validator_name == "validate_color":
+            return "string /* Color */"
+        if validator_name == "validate_gradient":
+            return "string /* Gradient */"
+        if validator_name == "fps_validator":
+            return "number /* FPS */"
+
+    # Standard Types
+    if validator == str:
```

Ruff (0.8.2)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)